packages/button/postcss.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
If we're going to have postcss.config.js in every package now, I'd try to inherit from some base config, similar to our approach with jest.config.js